cmd/abapEnvironmentCloneGitRepo_test.go

```diff
@@ -151,7 +173,7 @@ func TestCloneStepErrorMessages(t *testing.T) {
 
 		err := runAbapEnvironmentCloneGitRepo(&config, nil, &autils, client)
 		if assert.Error(t, err, "Expected error") {
-			assert.Equal(t, "Clone of '"+config.RepositoryName+"' with branch '"+config.BranchName+"' failed on the ABAP System", err.Error(), "Expected different error message")
+			assert.Equal(t, "Clone of '/DMO/REPO_A', branch 'branchA', commit 'ABCD1234' failed on the ABAP System", err.Error(), "Expected different error message")
```

Comment:
Looking at the full messages here: Maybe you could also add Software Component. So:
`Clone of Software Component '/DMO/REPO_A', branch 'branchA', commit 'ABCD1234' failed on the ABAP System`